packages/xo-server-sdn-controller/src/index.js

```diff
@@ -326,15 +326,15 @@ class SDNController extends EventEmitter {
 
   async _objectsUpdated(objects) {
     await Promise.all(
-      map(objects, async (object, id) => {
+      map(objects, object => {
```

Comment:
`id` is not needed in the core of the method, I've forgotten to remove it a few PRs ago ^^'